### ISSUE TITLE
CAST / Jira compatibility question

### ISSUE THREAD

**Issue by vincenzo-mazzotta** [CAST-Extend/com.castsoftware.uc.cast2jira#15]
Good afternoon,
 I'd like to know if the extension is compatible with Jira 8.13.2 and CAST 6.1.
 
 This question because we can't create issue in Jira to publish state of workflow during loading and elaborate on CASE.

Thanks a lot for your support

**Comment by NevinK** [comment]
No, the extension is not compatible with CAST 6.1.  To use it you will need to upgrade to 8.3.x
